message.go

```diff
@@ -122,7 +122,7 @@ type header struct {
 // from the given reader. The byte order is figured out from the first byte.
 // The possibly returned error can be an error of the underlying reader, an
 // InvalidMessageError or a FormatError.
-func DecodeMessage(rd io.Reader) (msg *Message, err error) {
+func DecodeMessage(rd io.Reader, fds []int) (msg *Message, err error) {
```

Comment:
Can we instead introduce this as DecodeMessageExtractingFDs and have the existing DecodeMessage function call that ?

Reply:
Absolutely! i'll write a new patch right away. thanks for looking at this!